setup.py

```diff
@@ -90,7 +90,6 @@ def read(path: str) -> str:
         "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: 3.12",
-        "Programming Language :: Python :: 3.13",
```

Comment:
this line was added by this PR https://github.com/crate/crate-operator/commit/21275f8336d6a5731c258289f6cb539c2a608a3c
I don't know if it's necessary to remove it.

Reply:
not sure, but I think it's fine to remove it for now since 3.13 is not (yet) supported.